python/tutorials/05-layer-norm.py

```diff
@@ -337,7 +337,7 @@ def test_layer_norm(M, N, dtype, eps=1e-5, device='cuda'):
         line_names=['Triton', 'Torch'] + (['Apex'] if HAS_APEX else []),
         styles=[('blue', '-'), ('green', '-'), ('orange', '-')],
         ylabel='GB/s',
-        plot_name='layer-norm-backward',
+        plot_name='layer-norm-forward',
```

Comment:
what is going on here

Reply:
i am simply correcting the name of the plot.

the benchmark is configured to run the forward pass but the plot is incorrectly labeled backward.

this has nothing to do with the PR but i thought i should correct it anyway.